react-app/ccf/src/pages/create-acc-menu/CreateAccMenu.css

```
@@ -2,17 +2,27 @@
     display: flex;
     align-items: center;
     justify-content: center;
-    height: 80vh;
+    min-height: 100vh;
+    width: 90vw;
+    max-width: 1200px;
+    margin: 0 auto;
     padding: 20px;
-    gap: 50px;
+    gap: 40px;
     font-family: 'Roboto', sans-serif;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)